src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
+const list = document.querySelectorAll('li');
+
+list.forEach(item => {
+  const span = document.createElement('span');
+
+  item.prepend(span);
+  span.prepend(span.nextSibling);
+});
+
+tree.addEventListener('click', (e) => {
+  const item = e.target.closest('span').nextSibling;
+
+  if (!item) {
```

Comment:
You can avoid using additional return if you will check vice versa: `if (item) { // do smth }`